src/frontend/src/test-e2e/util.ts

```diff
@@ -3,56 +3,57 @@ import { command } from "webdriver";
 import * as SeleniumStandalone from "selenium-standalone";
 import { ChildProcess } from "selenium-standalone";
 
+const MOBILE_SCREEN: ScreenConfiguration = {
+  screenType: "mobile",
+  windowSize: "360,640",
+};
+
+const DESKTOP_SCREEN: ScreenConfiguration = {
+  screenType: "desktop",
+  windowSize: "1920,1080",
+};
+
 export async function runInBrowser(
-  test: (browser: WebdriverIO.Browser) => Promise<void>
+  test: (
+    browser: WebdriverIO.Browser,
+    runConfig: RunConfiguration
+  ) => Promise<void>
 ): Promise<void> {
   await runInBrowserCommon(true, test);
 }
 
 export async function runInNestedBrowser(
-  test: (browser: WebdriverIO.Browser) => Promise<void>
+  test: (
+    browser: WebdriverIO.Browser,
+    runConfig: RunConfiguration
+  ) => Promise<void>
 ): Promise<void> {
   await runInBrowserCommon(false, test);
 }
 
-export async function startWebdriver(): Promise<ChildProcess | undefined> {
```

Comment:
why did this move?

Reply:
I found it confusing having that function in between the "runIn..." functions. But it'll be removed by my second PR anyways :-)